metadata/providers/package_metadata_info.bzl

```diff
@@ -2,24 +2,29 @@
 
 visibility("public")
 
-def _init(metadata, files = []):
+def _init(metadata, attributes = [], files = []):
     return {
+        "metadata": metadata,
+	"attributes": attributes,
         "files": depset(
             direct = [
                 metadata,
             ],
             transitive = files,
         ),
-        "metadata": metadata,
     }
 
 PackageMetadataInfo, _create = provider(
     doc = """
 Provider for declaring metadata about a Bazel package.
 
-> **Fields in this provider are not covered by the stability gurantee.**
+**Fields in this provider are not covered by the stability gurantee.**
 """.strip(),
     fields = {
+        "attributes": """
```

Comment:
I don't think this needs to be a depset. This should only be providers from `ctx.attr.attributes`, which is always a list.